proxy/README.md

```diff
@@ -0,0 +1,12 @@
+# Keycloak Security Proxy
+
+The Keycloak Security Proxy docker image.
+
+## Usage
+
+This image is intended to be used with an existing `/tmp/conf/proxy.json` on the host machine. 
+See [the proxy documentation](https://keycloak.gitbooks.io/documentation/server_installation/topics/proxy.html) 
+for more details.
```

Comment:
@jpkrohling should we add one previous step here like https://keycloak.gitbooks.io/documentation/server_installation/topics/network/https.html ? I don't think this image will run without a certificate.